src/components/SignIn/SignIn.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { BsFacebook, BsGoogle } from 'react-icons/bs';
+
+function SignIn() {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className="flex flex-col justify-center items-center" data-testid="sign-in">
+      <h1 className="text-5xl font-bold mb-10 text-primary pt-9">SIGN IN</h1>
+      <form onSubmit={handleSubmit} className="flex flex-col justify-center">
+        <label htmlFor="email" className="mt-3" >
```

Comment:
label and input are two separate tags, so nesting them is not good, add the label first and the input second, as they mentioned in the documentation https://www.w3schools.com/tags/tag_label.asp

Reply:
Yeah tell eslint to agree on that🤣 I did it like that first bitbeslint gave me an errorz after some Googling it accepted this version🙂

Reply:
yeah, the input must be inside the label according to eslint

Reply:
but it doesn't work, if you click on the label it doesn't focus on the input

Reply:
eslint is not always right about things

Reply:
I know and I am this 🤏 close to destroy eslint🙃

Reply:
believe me when eslint said this is incorrect be sure it's the correct answer 😂

Reply:
I also like to keep label outside the input, but it makes the code a bit more readable if you nest them. Hence, why eslint recommends to nest them. Id accept both but since you nested them let's move on with this.

Reply:
but @Iseluin if you click on the label the corresponding input doesn't trigger if you nesting the input

Reply:
doesnt trigger means it doesnt let you type ?